src/hashbang.sh

```diff
@@ -111,7 +111,9 @@ makekey() {
 
 # Fetch host data for later.
 # If this fails there is no point in proceeding
-host_data=$(curl -sH 'Accept:text/plain' https://hashbang.sh/server/stats)
```

Comment:
I'm guessing it's probably for fixing the line break bug, although this could be accomplished if we simply added an end of line when requesting the data. 

-----Original Message-----
From: "Lance R. Vick" <notifications@github.com>
Sent: ‎5/‎31/‎2015 2:47 PM
To: "hashbang/hashbang.sh" <hashbang.sh@noreply.github.com>
Cc: "Alissa" <alissasquared@gmail.com>
Subject: Re: [hashbang.sh] Improve POSIX compliance. (#44)

In src/hashbang.sh:
> @@ -111,7 +111,9 @@ makekey() {
>  
>  # Fetch host data for later.
>  # If this fails there is no point in proceeding
> -host_data=$(curl -sH 'Accept:text/plain' https://hashbang.sh/server/stats)
Do we really need a temp file here? What is the incompatibility?
—
Reply to this email directly or view it on GitHub.